ibtk/src/utilities/IBTK_MPI.cpp

```diff
@@ -35,12 +36,14 @@ IBTK_MPI::setCommunicator(IBTK_MPI::comm communicator)
 IBTK_MPI::comm
 IBTK_MPI::getCommunicator()
 {
+    IBTKInit::check_initialized();
```

Comment:
This set of changes seems overly invasive - lets just set the default communicator to `MPI_COMM_NULL` and let MPI crash if we make any calls with that.